content/SCALE/SCALEUIReference/Virtualization/VirtualizationScreens.md

```diff
@@ -66,11 +66,11 @@ The **CPU and Memory** configuration wizard screen settings specify the number o
 | **Cores** | Required. Enter the number of cores per virtual CPU socket. The product of vCPUs, cores, and threads must not exceed 16. |
 | **Threads** | Required. Enter the number of threads per core. A single CPU core can have up to two threads per core. A dual core could have up to four threads. The product of vCPUs, cores, and threads must not exceed 16. |
 | **Optional: CPU Set (Examples: 0-3,8-11)** | Specify the logical cores that VM is allowed to use. Better cache locality can be achieved by setting CPU set base on CPU topology. E.g. to assign cores: 0,1,2,5,9,10,11 you can write: `1-2,5,9-11` |
-| **Pin vcpus** | When number of vcpus is equal to number of cpus in CPU set vcpus can be automatically pinned into CPU set. Pinning is done by mapping each vcpu into single cpu number in following the order in CPU set. This will improve CPU cache locality and can reduce possible stutter in GPU passthrough VMs. |
+| **Pin vcpus** | When number of vcpus is equal to number of cpus in CPU set vcpus can be automatically pinned into CPU set. Pinning is done by mapping each vcpu into single cpu number in following the order in CPU set. This improves CPU cache locality and can reduce possible stutter in GPU passthrough VMs. |
```

Comment:
```suggestion
| **Pin vcpus** | When the number of vCPUs equals the number of CPUs in **CPU Set**, vCPUs can be automatically pinned into **CPU Set**. Pinning is done by mapping each vCPU into a single CPU number following the order in **CPU Set**. This improves CPU cache locality and can reduce possible stutter in GPU passthrough VMs. |
```

Reply:
Goodness, the UI tooltips are messy.

Reply:
Yes they are! We need to start rewriting them so they make more sense!